<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>721</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label8">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>200</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label11">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>11</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label12">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>340</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>12</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label13">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>13</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label14">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>14</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label15">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>15</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label16">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>480</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>16</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>221</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="score2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="score1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>score</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restartButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>181</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>QUIT</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>601</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label7</zorder>
   <zorder>label5</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>label2</zorder>
   <zorder>label6</zorder>
   <zorder>label3</zorder>
   <zorder>label11</zorder>
   <zorder>label9</zorder>
   <zorder>label15</zorder>
   <zorder>label13</zorder>
   <zorder>label16</zorder>
   <zorder>label4</zorder>
   <zorder>label8</zorder>
   <zorder>label10</zorder>
   <zorder>label14</zorder>
   <zorder>label12</zorder>
   <zorder>label1</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>restart()</slot>
 </slots>
</ui>
